cirq/devices/grid_qubit_test.py

```diff
@@ -79,7 +79,7 @@ def test_diagram():
 -----KL-----
 """
     assert (cirq.GridQubit.from_diagram(s) ==
-            cirq.google.known_devices._parse_device(s)[0])
+            cirq.google.devices.known_devices._parse_device(s)[0])
```

Comment:
Consider exporting this `_parse_device` function on `cirq.google.devices`. We could rename it to `parse_device_ascii` or something like that.

Reply:
I just decided to change this test instead.  I am not sure why it was depending on this function.  It seemed like a bad test.